C/CImPlot/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "CImPlot"
+version = v"1.76.0"
```

Comment:
This version should be the one released [here](https://github.com/cimgui/cimplot/releases). As there is no stable release yet, let's keep this to v"0.0.0" for now.

Reply:
Maybe better to use 0.0.1?